VariantEncoding.md

```diff
@@ -94,7 +112,7 @@ Each `offset` is a little-endian value of `offset_size` bytes, and represents th
 The first `offset` value will always be `0`, and the last `offset` value will always be the total length of `bytes`.
 The last part of the metadata is `bytes`, which stores all the string values in the dictionary.
 
-## Metadata encoding grammar
+### Metadata encoding grammar
```

Comment:
I've updated these to not use more than one H1, which can cause issues with TOC. Pages should have just one H1.

Reply:
TIL as well